requirements.txt

```diff
@@ -1,26 +1,27 @@
-aiofiles
-aiohttp
-altair>=4.2.0
+aiofiles>=22.0,<24.0
+aiohttp~=3.0
+altair>=4.2.0,<6.0
 fastapi
 ffmpy
 gradio_client>=0.2.7
 httpx
 huggingface_hub>=0.14.0
-Jinja2
+Jinja2<4.0
 markdown-it-py[linkify]>=2.0.0
-pygments>=2.12.0
```

Comment:
Makes sense. Should have been deleted as part of https://github.com/gradio-app/gradio/pull/4150